libraries/Microsoft.Bot.Builder.AI.QnA/Utils/HttpRequestUtils.cs

```diff
@@ -70,6 +70,7 @@ public async Task<HttpResponseMessage> ExecuteHttpRequestAsync(string requestUrl
         private void SetHeaders(HttpRequestMessage request, QnAMakerEndpoint endpoint)
         {
             request.Headers.Add("Authorization", $"EndpointKey {endpoint.EndpointKey}");
+            request.Headers.Add("Ocp-Apim-Subscription-Key", $"EndpointKey {endpoint.EndpointKey}"); 
             request.Headers.UserAgent.Add(botBuilderInfo);
```

Comment:
Does this also not need a change to GetHostName? Its hard to view the E2E workflow without seeing all related changes in the same PR as well as a sample. Can we add them here? Also what changes are required to JS SDK?